swan-lake/development-tutorials/source-code-dependencies/organize-ballerina-code.md

```diff
@@ -112,7 +111,6 @@ The package structure after adding a non-default module will have the directory
 ├── main.bal
 └── modules
     └── util
-        ├── Module.md
```

Comment:
Don't we need `README.md` here?

Reply:
Previously, Module.md was auto generated. Now the README.md at the module level is not created automatically with `bal add` command.